resources/migrations/instance_analytics_views/h2-indexes.sql

```diff
@@ -0,0 +1,7 @@
+drop index if exists idx_view_log_timestamp;
+
+create index if not exists idx_view_model_id
+    on view_log(model_id);
```

Comment:
`idx_view_log_timestamp` already existed as an index on `model_id` — I think it was just named incorrectly. So I've added back `idx_view_log_model_id` with the fixed name